frontend/src/metabase/admin/datamodel/metadata/components/MetadataTableList/MetadataTableList.styled.tsx

```diff
@@ -13,3 +15,17 @@ export const AdminListItem = styled.a<AdminListItemProps>`
       opacity: 0.4;
     `};
 `;
+
+export const BackIconContainer = styled.span`
+  color: ${color("brand")};
+  cursor: pointer;
+`;
+
+export const HideIconButton = styled(IconButtonWrapper)`
+  float: right;
+  cursor: ${props => props.disabled && "not-allowed"};
+
+  &:hover {
+    color: ${color("brand")};
```

Comment:
Previously hover effect was not applied when `disabled`: https://github.com/metabase/metabase/pull/32514/commits/39c74040aab2af33545e0415774729beac35943c#diff-941767a59c0d72015ea9fcdadf56b339bce3c91b453ce55ec0a75bf7f74be5f3L318

Reply:
Great catch!